### ISSUE TITLE
Automatically adding logs won't work in Java and other apps

### ISSUE THREAD

**Issue by thesteve0** [logentries/le_openshift#10]
It seems that in the post_deploy action hook you are looking for files with *_log.* to follow. This won't work for a lot of different cartridge types. It is too specific to apache logs. I think it would just be better to add *.log or add nothing and let the user add it in the ini file.

**Comment by robertoestivill** [comment]
Where are you seing that wildcard? 
Also, is `LOG_ROOT=/var/log` right? shouldn't be `$OPENSHIFT_LOG_DIR` ?

**Comment by andrea-sdl** [comment]
@thesteve0 that's right. You have to change the post_deploy script to follow also the filename pattern you want.

I made some changes to use it on my node.js app, try the https://raw.githubusercontent.com/andrea-sdl/le_openshift/master/.openshift/action_hooks/pre_start and let me know if it helps.

**Comment by StephenHynes7** [comment]
@andrea-sdl if it works maybe we can get a PR. This is an incredibly old lib which has sat around gathering dust so we would love to get some contribution love if we can :)

**Comment by andrea-sdl** [comment]
@StephenHynes7 ok! Did the pull request ;) lemme know if it's ok or if I should change something ;) hope it helps.
